Feature/numeric stepper props (#15)
vtex-apps/product-quantity: docs/README.md

| `warningQuantityThreshold` | `Number` | Displays the quantity of remaining items in stock if the available quantity is less than or equal to the value given to this property. Default: 0 (does not appear). | `0` |
| `size` | `NumericSize`| Preset values `font-size` and `padding` to the component | `'small'` |
| `showLabel` | `boolean` | If it should show a label | `true` |

> | `showMeasurementUnit` | `boolean` | If it should show a measumentUnit | `true` |

Reviewer: ```suggestion
| `showMeasurementUnit` | `boolean` | If it should show a measumentUnit | `false` |
```

> | `showMeasurementUnit` | `boolean` | If it should show a measumentUnit | `false` |

Reviewer 2: Props  `unitMultiplier` and `measurementUnit` are missing here

Reviewer 2: furthermore, is there any case where `measurementUnit` is present but `showMeasurementUnit` is false? Perhaps the unit should just be always displayed when `measurementUnit` is defined, and you could remove the `showMeasurementUnit` prop

Reviewer 2: (nevermind these comments, the changelog was misleading)

> | `warningQuantityThreshold` | `Number` | Displays the quantity of remaining items in stock if the available quantity is less than or equal to the value given to this property. Default: 0 (does not appear). | `0` |
> | `size` | `NumericSize`| Preset values `font-size` and `padding` to the component | `'small'` |
> | `showLabel` | `boolean` | If it should show a label | `true` |
> | `showMeasurementUnit` | `boolean` | If it should show a measumentUnit | `false` |

Reviewer 3: ```suggestion
| `warningQuantityThreshold` | `number` | Displays the number of remaining items in stock if the total amount is less than or equal to the number configured in this prop. | `0` |
| `size` | `enum`| Preset values for `font-size` and `padding` to the component. Possible values are `small`, `regular` (medium size) and `large`. | `small` |
| `showLabel` | `boolean` | Defines whether a label is shown (`true`) or not (`false`). | `true` |
| `showMeasurementUnit` | `boolean` | Defines whether a measurement unit is shown (`true`) or not (`false`). | `false` |
```

> ### NumericSize
> 

Reviewer 3: ```suggestion
```

Reviewer 3: This whole section (`NumericSize`) should be removed from documentation.

<!-- prettier-ignore-end -->
<!-- ALL-CONTRIBUTORS-LIST:END -->

> This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!

Reviewer 3: ```suggestion
This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind are welcome!
```
